### DISCUSSION TITLE
Option for minimessage anvil colors

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#1131] [__Completed__] [answered]
It would be nice to have the option to use minimessage tags in anvil colors, like `<gradient:aqua:dark_aqua>Item Name` or something

**Comment by emielderckx** [comment, marked as answer]
Implemented in [aafe275](https://github.com/PurpurMC/Purpur/commit/aafe27514049451fc213bafcdf7d2a252f94f81d).
